_includes/announcement-blog.html

```diff
@@ -0,0 +1,69 @@
+{% assign cohort = include.cohort | strip %}
+{% assign people = site.data.people %}
+{% assign cohortData = site.data.cohorts[cohort] %}
```

Comment:
Could you use `_` and lower case in all variable names instead of camel case? Thanks

Reply:
Yes, please. Best for consistency. Done...